include/swift/Basic/SourceManager.h

```diff
@@ -23,10 +23,16 @@
 
 namespace swift {
 
+inline llvm::IntrusiveRefCntPtr<llvm::vfs::OverlayFileSystem>
```

Comment:
Nitpick: This should still be `static`, or it can result in symbol visibility errors if it's ever *not* inlined.